staging/kos/pkg/controllers/connectionagent/network_interface.go

```diff
@@ -194,16 +194,28 @@ func (ca *ConnectionAgent) createLocalNetworkInterface(att *netv1a1.NetworkAttac
 	}
 	statusErrs = ca.launchCommand(parse.AttNSN(att), ifc.LocalNetIfc, att.Spec.PostCreateExec, ifc.postCreateExecReport.Store, "postCreate", true)
 	if att.Status.IfcName == "" {
+		lastClientWrName := att.LastClientWrite.Name
+		lastClientWrTime := att.LastClientWrite.Time.Time
```

Comment:
After this PR is merged or closed I propose a PR to make all controllers' code switch from `time.Time` to `k8smetav1.MicroTime`  to avoid the `.Time.Time` goofiness.